ynput/ayon-kitsu
# Status change on user action

## App start status change
Now we have ability to change status on users starting app action.

![1-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/4d60ed68-dce4-4926-8689-8b617021305e)

## Pause other tasks with WIP status
We can turn off pausing all other WIP tasks. This logic made of state: only one task can be WIP at a time.

![2-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/9cf448ca-8ca1-49c9-9861-ee1a2af8d4a0)


## Farm render status change
Previosly task was getting status change (WFA) only at publishing state in deadline so in between of publishing job to farm and publishing to kitsu we have nothing indicating current task status. Now we have FARM status change when farm rendering is on

![3-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/c3ba2ea9-8037-4113-8593-35088467af39)




## Additional info
 The settings allow to configure status change conditions so statuses will not double. And some other changes to settings needed to be done to make all logic work and ahve opportunity to swith in on or off 
![image](https://github.com/user-attachments/assets/121e078b-91d0-455f-bbba-f62516738426)



## Testing notes:

1. Add nessesary statuses to your kitsu production
2. configure status changes in addon
3. mark some statuses in kitsu as WIP (only for test)
4. start other task
5. see all other tasks now was PAUSED
6. publish a job on farm
7. see status change to FARM status
